docs/doxygen/config.doxyfile

```diff
@@ -851,8 +851,9 @@ INPUT                  = ./docs/doxygen \
                          ./docs/doxygen/demos \
                          ./docs/doxygen/migration \
                          ./docs/doxygen/developer \
-                         ./docs/doxygen/logging \
                          ./platform/include \
+                         ./platform/posix
+                         ./docs/doxygen/logging \
                          ./demos/logging-stack
```

Comment:
Let's just add `\` to the end, so when we make changes we only need to change one line, not two
```suggestion
                         ./demos/logging-stack \
```